sandbox.html

```diff
@@ -474,9 +474,33 @@
     <label for="embeddingTargetId">Embedding target ID of the map view *</label>
     <input name="embeddingTargetId" id="embeddingTargetId" required>
   </div>
+  <div>
+    <label for="geometry">Additional layers (GeoJSON)</label>
+    <textarea name="additionalLayers" id="additionalLayers" rows="10" placeholder="featureCollections array"></textarea>
```

Comment:
These placeholders are not helpful, they just duplicate the labels. I'd remove them.